doc/POST_INSTALL.md

```diff
@@ -0,0 +1,7 @@
+Once the installation is complete, the app will need to complete the registration process by visiting https://__DOMAIN____PATH__.
+
+Details for MySQL database to be enterted while registration process:
+
+Database login:    __DB_USER__
+Database name:     __DB_NAME__
+Database password: __DB_PWD__
```

Comment:
```suggestion
- Database login:    __DB_USER__
- Database name:     __DB_NAME__
- Database password: __DB_PWD__
```

I suspect that, when rendered as markdown, these will be dispayed as concatenated on the same line (otheriwse you'd have to have an empty line between each of those) as can be seen by looking at https://github.com/YunoHost-Apps/matomo_ynh/blob/2ddee462c37654ff9079fb9df67f2dd7dd1199ec/doc/POST_INSTALL.md on Github ...

So probably better to have them as item list